docs/concepts.rst

```diff
@@ -116,17 +116,21 @@ DAGs can be used as context managers to automatically assign new operators to th
 
     op.dag is dag # True
 
-.. _concepts:functional_dags:
+.. _concepts:decorated_flows:
 
-Functional DAGs
+Decorated Flows
 ---------------
 
-DAGs can be defined using functional abstractions. Outputs and inputs are sent between tasks using
-:ref:`XCom values <concepts:xcom>`. In addition, you can wrap functions as tasks using the
-:ref:`task decorator <concepts:task_decorator>`. Airflow will also automatically add dependencies between
-tasks to ensure that XCom messages are available when operators are executed.
+.. versionadded:: 2.0.0
 
-Example DAG with functional abstraction
+Airflow 2.0 adds a new style of authoring dags called Decorated Flows which removes a lot of the boilerplate
```

Comment:
Should we add something like `(aka functional DAGs)` just to point people who already know about it that this is the same thing?

Reply:
Good idea